AudioKitSynthOne/DSP/Kernel/S1DSPKernel.mm

```diff
@@ -44,6 +44,11 @@
     sp_phaser_init(sp, phaser0);
     sp_port_create(&monoFrequencyPort);
     sp_port_init(sp, monoFrequencyPort, 0.05f);
+    sp_port_create(&lfo1Port);
+    const float lfoSmoothHalftime = 0.0053125f; // empirical
```

Comment:
I'd make it `static const` and prefix const variables with a `k`.. ie `kLfoSmoothHalftime` (so everyone knows what its good for/not meant to be changed/used as a magic number)

Reply:
done